Samples/StorageProviders/StorageProviders/BaseJSONStorageProvider.cs

```diff
@@ -15,6 +17,13 @@ namespace Samples.StorageProviders
     /// </summary>
     public abstract class BaseJSONStorageProvider : IStorageProvider
```

Comment:
@jdom Agree. What about including the JSON & MongoDB provides as part of Orleans with the other storage providers? That way the tests & samples could reference the same class.